data/json/items/ammo/500.json

```
@@ -26,7 +26,7 @@
     "id": "bp_500_Magnum",
     "copy-from": "500_Magnum",
     "type": "AMMO",
-    "name": { "str": ".500 S&W Magnum, black powder" },
+    "name": { "str": ".500 S&W Magnum, black powder", "str_pl": ".500 S&W Magnums, black powder" },
```

```suggestion
    "name": { "str_sp": ".500 S&W Magnum, black powder" },
```
"Magnums" are only used in referring to guns firing the ammunition.